src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+
+    private double sideA;
+    private double sideB;
+    private String color;
+
+    public Rectangle(double sideOne, double sideTwo, String color) {
+        this.sideA = sideOne;
+        this.sideB = sideTwo;
+        this.color = color;
+    }
+
+    public void getSides() {
+        System.out.println("Side one : " + sideA + ", side two : " + sideB);
+    }
+
+    @Override
+    public double getArea() {
+        return sideA * sideB;
+    }
+
+    @Override
+    public String getColor() {
```

Comment:
remove this method everywhere exception Figure class

Reply:
Done